### PR TITLE
Added a mentor

### PR DESCRIPTION
**Pull request by Hackerbone**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link](https://hackerbone.github.io/gci18.fossasia.org/)

## Description / Changes
Added mentor Mohit Kumar.

## Screenshots if any:
screenshot 2018-11-14 at 3 08 31 pm

- - - - - - - - - - - -

### CODE REVIEW

File: _data/mentors.yml [hunk 1]
```diff
@@ -74,10 +74,14 @@
...
   lng: 76.779419
+  
+- name: Mohit Kumar
+  image: mohitkumar.jpeg
+  github: batbrain7
 
 - name: Nicco Kunzmann
   image: Nicco Kunzmann.png
-  github: niccokunzmann
+  github: n====iccokunzmann
```
Comment (nikit19) [inline _data/mentors.yml]: why?

Comment (Hackerbone) [reply _data/mentors.yml]: Oh okay ill resolve it..

**Comment by Hackerbone** [comment]
I'll check out maybe the merge conflict did that... Ill correct it.

**Comment by Hackerbone** [comment]
I'll open a PR again seems to  have some problems.

**Pull request closed without merging**